sstables/leveled_manifest.hh

```diff
@@ -49,6 +49,7 @@
 #include <boost/range/algorithm/partial_sort.hpp>
 
 class leveled_manifest {
+    table& _table;
```

Comment:
This is de-modularizing. We now have compaction manifests knowing about tables. This is spaghettifing the code.

I'll let it go this time but I'm sure I'll come to regret it as someone will have to disentangle it.

Reply:
That sucks indeed. However I think that making the schema be aware of whether or not we're bootstrapping would be much worse.

Hopefully we'll be able to revert this entire patch soon, so nobody will have to disentangle anything.